### DISCUSSION TITLE
2006/07/03/labview-%E6%98%AF%E7%BC%96%E8%AF%91%E5%9E%8B%E8%AF%AD%E8%A8%80%E8%BF%98%E6%98%AF%E8%A7%A3%E9%87%8A%E5%9E%8B%E8%AF%AD%E8%A8%80

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#89] [Announcements]
# 2006/07/03/labview-%E6%98%AF%E7%BC%96%E8%AF%91%E5%9E%8B%E8%AF%AD%E8%A8%80%E8%BF%98%E6%98%AF%E8%A7%A3%E9%87%8A%E5%9E%8B%E8%AF%AD%E8%A8%80

https://lv.qizhen.xyz/optimization\mechanism

https://blog.qizhen.xyz/2006/07/03/labview-%E6%98%AF%E7%BC%96%E8%AF%91%E5%9E%8B%E8%AF%AD%E8%A8%80%E8%BF%98%E6%98%AF%E8%A7%A3%E9%87%8A%E5%9E%8B%E8%AF%AD%E8%A8%80

**Comment by ruanqizhen** [comment]
之前博客下的留言：


基帆
2010年03月11日 11:44 编辑
用着labVIEW感觉就像解释语言…顶大侠~

回复
sillyball
2006年10月12日 09:30 编辑
如果是编写测试测量、控制、仿真等领域的应用程序，LabVIEW 比 C 的运行效率和开发效率都高。因为 LabVIEW 特别为这些领域做了优化….
开发效率高是真的,运行效率高就未必了

回复
Unknown
2006年09月26日 16:14 编辑
曾经同NI的技术支持关于LabVIEW参考书问题打听过，他们的见解是目前市面上的LabVIEW参考书最新版的也只是7.1的，他们的推荐是杨乐平主编的《LabVIEW程序设计与应用》与《LabVIEW高级程序设计》。

回复
Qizhen
2006年07月6日 19:13 编辑
我以前看的书比较少，觉得 NI 公司自己编写的教程是最好的。
不过，我现在也打算买些 LabVIEW 的书看看别人是如何理解它的。
在 LabVIEW 论坛上打听打听会有读者来推荐的。

回复
yi
2006年07月6日 00:51 编辑
谢谢大牛的耐心解答。明白了很多。
对大侠的敬仰有如滔滔江水（此处500字省略。。。）
自己开始学习一些关于数据采集的东东，用到是NI的采集卡（像PXI-4472,6062E,6023E等）。感觉很深奥哦。还需要进一步学习labVIEW和信号处理的知识阿！

回复
晓元
2006年07月5日 17:26 编辑
搭车同问阮大侠,
市面上有没有那种知其然更知其所以然的LabVIEW书籍？

回复
Qizhen
2006年07月5日 12:52 编辑
回答问题 :)
1. LabVIEW 一般来说是靠牺牲运行效率来提高开发效率。所以某些 LabVIEW 程序运行效率会比 C 语言低。不过 C 语言运行效率比汇编低，选用哪种语言要综合考虑。但是，如果是编写测试测量、控制、仿真等领域的应用程序，LabVIEW 比 C 的运行效率和开发效率都高。因为 LabVIEW 特别为这些领域做了优化。
 
2. LabVIEW 适合大型程序开发，比如我刚做完的一个项目由上千个 VI 组成。应该算是比较大的了，用 LabVIEW 做完全没有问题。
 
3. 对 NI 的培训我一无所知，要打电话问相关的人员了。

回复
yi
2006年07月5日 00:59 编辑
有些问题想请教一下这位大侠^_^
1.是不是labVIEW的执行效率会低些呢（与C比较）？
2.labVIEW适合开发大型程序吗？
3.另外关于NI培训的问题：在NI网站上没有看到培训的费用信息，不过我猜想很贵。能否透露一二？
谢谢。

回复
永青
2006年07月3日 17:49 编辑
阮大侠高屋建瓴，如庖丁解牛般解决了一个长久以来一直困惑我的问题，pfpf。

回复
